runestone/activecode/js/activecode.js

```diff
@@ -774,7 +775,16 @@ errorText.NotImplementedError = $.i18n("msg_activecode_not_implemented_error");
 errorText.NotImplementedErrorFix = $.i18n("msg_activecode_not_implemented_error_fix");
 
 
-
+ActiveCode.prototype.addJSONLibrary = function () {
+    Sk.externalLibraries = {
```

Comment:
This should check to see if there are already other externalLibraries specified elsewhere.  Otherwise this will wipe them out.